capytaine/io/xarray.py

```diff
@@ -257,15 +257,39 @@ def assemble_dataset(results: Sequence[LinearPotentialFlowResult],
     """
     dataset = xr.Dataset()
 
+    error = 'results must be either of type LinearPotentialFlowResult or a bemio.io object'
+    if hasattr(results, '__iter__'):
+        try:
+            if 'capytaine' in results[0].__module__:
+                bemio_import = False
+            else:
+                raise error
```

Comment:
You forgot the exception, you are raising only the string of the error message.

Reply:
Thanks, this is fixed in 3be53fa.